.nvmrc

```diff
@@ -1 +1 @@
-v14.9.0
+v14.17.0
```

Comment:
Is there any reason to change node version?

Reply:
I was stuck with an error stating that a dependency required node >= 14.17 during my first `yarn install`

Reply:
Alright, then let's leave it in 14.17 then